crates/ibc/src/core/ics04_channel/handler/chan_open_try.rs

```diff
@@ -48,6 +48,10 @@ where
     {
         let client_id_on_b = conn_end_on_b.client_id();
         let client_state_of_a_on_b = ctx_b.client_state(client_id_on_b)?;
+
+        client_state_of_a_on_b.assert_not_frozen()?;
```

Comment:
Similarly, the `chan_open_init`, `chan_close_init` and `chan_close_confirm` need an `assert_not_frozen` check as well

Reply:
note that using a `get_client_state_validation()` would solve this issue

Reply:
Good catch 👌🏻 
These were missed before and it came to light